File: my-folder/0383-ransom-note/solution.js
/**
 * @param {string} ransomNote
 * @param {string} magazine
 * @return {boolean}
 */
var canConstruct = function(ransomNote, magazine) {

    for(const i of magazine){
        ransomNote = ransomNote.replace(i, "")
    }

    if(!ransomNote) return true;

    return false

    
};

File: my-folder/0383-ransom-note/solution.rs
impl Solution {
    pub fn can_construct(ransom_note: String, magazine: String) -> bool {
        let mut tempRansom = ransom_note;
        for c in magazine.chars(){
           if let Some(pos) = tempRansom.find(c){
            tempRansom.remove(pos);
           }
        }

       tempRansom.is_empty()
    }
}
